app/controllers/login_controller.rb

```diff
@@ -0,0 +1,21 @@
+class LoginController < ApplicationController
+  def index
+  end
+
+  def create
+    user = User.find_by(email: params[:email]).try(:authenticate, params[:password])
+
+    if user
+      session[:uuid] = user.uuid
+      session[:email] = user.email
```

Comment:
I would store just the user ID in the session. The smaller your cookies, the better.